decentraland/unity-explorer
# feat: Procedural Landscape

# Changes 

Replaced empty parcels with procedurally generated terrain

# How to test

- Wait for the loading screen to finish.
- Check that the empty parcels have the procedurally generated content.
- Please check that no asset from the terrain overlaps with owned scenes (trees spawning near the edge of the empty parcels and their branches/leaves are over-owned parcel areas) you don't need to check 100% of Genesis City.
